### DISCUSSION TITLE
Commandline/Unattended Mode

### DISCUSSION THREAD

**Discussion by JJDE** [DaxStudio/DaxStudio#503] [Ideas]
would be very handy if daxstudio.exe could process commandline parameters.  for example we have the need for a scheduled export from pbi dataset data to csv. think of "DaxStudio.exe" -connection xy -queryfile myquery.dax -output excel -outputpath "C:\temp\...xlsx"

**Comment by Ferraglia** [comment]
Hello DaxStudio Team! This is really needed. Hope you can prioritize the functionality. Thanks

**Comment by dgosbell** [comment]
To push the results of a DAX query to CSV it's probably simpler and lighter weight to use a small PowerShell script like the following [example script](https://gist.github.com/dgosbell/75a4c2b6c1eae6ba1a8f4fdddd995119).
